vegafusion-python/tests/specs/dual_axis.json

```diff
@@ -0,0 +1,294 @@
+{
+    "layer": [
+        {
+            "layer": [
+                {
+                    "layer": [
+                        {
+                            "mark": {
+                                "clip": true,
+                                "type": "bar",
+                                "color": "#2266D3",
+                                "tooltip": false
+                            },
+                            "encoding": {
+                                "x": {
+                                    "sort": {
+                                        "op": "sum",
+                                        "field": "order_total",
+                                        "order": "descending"
+                                    },
+                                    "type": "nominal",
+                                    "field": "product_category",
+                                    "scale": {
+                                        "type": "linear"
```

Comment:
Both X-axis encodings use nominal fields but specify linear scale type, which is invalid per Vega-Lite schema.

Apply to both instances:
```
-                                    "scale": {
-                                        "type": "linear"
-                                    }
+                                    "scale": {
+                                        "type": "band"  
+                                    }
```
Also applies to: 154-156